pkg/oci/client/client_test.go

```diff
@@ -54,3 +56,90 @@ func TestInstanceTerminalState(t *testing.T) {
 		})
 	}
 }
+
+type mockComputeClient struct{}
+
+type mockVirtualNetworkClient struct{}
+
+type mockLoadBalancerClient struct{}
+
+func TestRateLimiting(t *testing.T) {
```

Comment:
Not sure i see much value in this test as what's being tested is the kubernetes code from the flowcontrol pkg